src/App.js

```diff
@@ -1,12 +1,53 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import Pokemon from './components/pokemon/Pokemon';
+import axios from "axios";
+import Button from "./components/button/Button";
+import logo from "../src/assets/International_Pokémon_logo.svg.png"
 
-function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+function App(){
+   const [pokemon, setPokemon] = useState({});
```

Comment:
Als je een makkelijk onderscheid wil hebben tussen of het al wél of niet is ingeladen, zou ik hier voor default waarde van `undefined` gaan
```suggestion
   const [pokemon, setPokemon] = useState();
```